### PR TITLE
I completed my CSS styling!  For now...

### PR DESCRIPTION
**Pull request by wiggitywhitney**

### CODE REVIEW

File: public/styles.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    font-weight: 700;
+    src: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_bd-46b91bda68161c14e554a779643ef4957431987b._V2_.woff2) format("woff2"),url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_bd-b605252f87b8b3df5ae206596dac0938fc5888bc._V2_.woff) format("woff")}
+    
+@font-face {font-family: 'Amazon Ember';
+    font-style: italic;
+    font-weight: 700;
+    src: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_bdit-80ff7aba37dd1ff5a6b90233a19e3a780a96dc2f._V2_.woff2) format("woff2"),url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_bdit-57598ce426a612be5a1d15eee08252668fca5e7a._V2_.woff) format("woff")}
+    
+.a-ember body {
+    font-family:"Amazon Ember", Arial, sans-serif}
```
Comment (ArohanD) [inline public/styles.css]: suggest: move curlys to new line

File: public/styles.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    
+.a-ember
+.a-text-quote {
+    font-family: "Amazon Ember", Arial, sans-serif}
```
Comment (ArohanD) [inline public/styles.css]: suggest: move curlys to new line

File: public/styles.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+
+#des_itemDescriptionModule {
+    display: flex;
+    width: 31%;
```
Comment (ArohanD) [inline public/styles.css]: pls check and see if the page is affected accordingly when you shrink the width of the browser window

File: public/styles.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 5px;
+}
+
+#des_title {
+    font-family: "Amazon Ember";
+    font-size: 21px!important;
```
Comment (ArohanD) [inline public/styles.css]: Please remove !important as it may override other styling

File: public/styles.css [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    line-height: 1.3!important;
```
Comment (ArohanD) [inline public/styles.css]: Please remove !important as it may override other styling

File: public/styles.css [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+#des_numOfQuestions {
+    font-family: "Amazon Ember";
+    color: royalblue;
+    font-size: 15px;
+}
+
+#des_listPrice {
+    font-family: "Amazon Ember";
```
Comment (ArohanD) [inline public/styles.css]: You have font family declared in a lot of places, you only need to declare it once in the global wrapper. As per style guide, please refactor

**Comment by ArohanD** [comment]
Great job! A couple of important (lol) comments to look into before your next request

File: public/styles.css (hunk 1 above)
Comment (wiggitywhitney) [reply public/styles.css]: Done!

File: public/styles.css (hunk 2 above)
Comment (wiggitywhitney) [reply public/styles.css]: Done!

**Pull request merged**